<launch>
    <arg name="device" default="plughw:1,0" />
  
    <!-- publish audio data as wav format -->
    <node name="audio_capture" pkg="vz_acoustic_scene_analysis" type="audio_capture" output="screen">
      <param name="format" value="wave" />
      <param name="channels" value="1" />
      <param name="depth" value="16" />
      <param name="sample_rate" value="44100" />
      <param name="device" value="$(arg device)" />
    </node>
    <node pkg="vz_acoustic_scene_analysis" type="ros_interface.py" name="ros_interface" output="screen" />
  </launch>